### ISSUE TITLE
Carla laplacian checkpoint

### ISSUE THREAD

**Issue by Andyyh00** [theialab/noksr#1]
Hi,

The results look very impressive! I've noticed at the inference that you have mentioned the `Carla_Laplacian_best.ckpt` file. However, the pre-trained checkpoints only provided pre-trained weights for the serial and KNN models. I was wondering if these weights were already trained, including the Laplacian loss. 

If not, I was wondering when you will be releasing this checkpoint?

**Comment by colinzhenli** [comment]
Hi,

Thanks for your reminder! We forgot to update the checkpoint files.

I've updated the link — it should now include the pretrained model with Laplacian loss, let me know if there is any other issue.

**Comment by Andyyh00** [comment]
Hi, Thank you for updating
